python/unitytrainers/ppo/trainer.py

```diff
@@ -43,6 +43,7 @@ def __init__(self, sess, env, brain_name, trainer_parameters, training, seed):
         self.use_recurrent = trainer_parameters["use_recurrent"]
         self.use_curiosity = bool(trainer_parameters['use_curiosity'])
         self.sequence_length = 1
+        self.has_updated = False
```

Comment:
Can you put a comment describing what it does ?
I think it could also benefit a better name. From the code, I think it is something like `had_first_update`